ui/desktop/src/components/settings/Settings.tsx

```diff
@@ -37,13 +37,59 @@ const DEFAULT_SETTINGS: SettingsType = {
   extensions: [],
 };
 
+const BUILT_IN_EXTENSIONS = [
```

Comment:
nit: this might make this PR a bit more difficult but we might want developer enabled by default and in this list, so it can be toggled off? 

e.g. it might be weird to have developer enabled with non-developer also enabled

Reply:
actually was intended to stack them both on (wouldn't want just non-developer)

Reply:
(as it can make use of it - its more when all the tools aren't available) so that is ok (maybe we do want developer turned off but probably not yet)